dhcpv4/ztpv4/ztp.go

```diff
@@ -15,14 +16,39 @@ type VendorData struct {
 
 var errVendorOptionMalformed = errors.New("malformed vendor option")
 
+const entIDCiscoSystems = 0x9
```

Comment:
Please move this to the iana package.

https://www.iana.org/assignments/enterprise-numbers/enterprise-numbers